virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,18 +1,26 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to an external network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+{VirtProductName} is installed with a default internal `pod` network. You update your xref:../../../virt/node_network/virt-updating-node-network-config.adoc#virt-about-nmstate_virt-updating-node-network-config[node networking configuration] depending on the type of network to which you want to connect.
+
+[IMPORTANT]
+====
+When connecting to an external network, it is essential to first create a _Linux bridge_. Until you create a Linux bridge, devices such as network interface card (NICs) are not available because your external network is not yet exposed. To expose your external network and make such devices available, you first need to create a Linux bridge.
+====
+
+Once a network bridge is created, you can start virtual machines on the external network in the same manner that you start virtual machines attached to internal networks, by using PXE booting, for example.
+
+To attach virtual machines to additional networks, you must configure a Linux bridge.
 
 To attach a virtual machine to an additional network:
 
 . Configure a Linux bridge.
```

Comment:
Actually, this is two steps:
1. Create a NodeNetworkConfigurationPolicy (Also, I'm not sure why you include the procedure when it's covered [elsewhere](https://deploy-preview-44495--osdocs.netlify.app/openshift-enterprise/latest/virt/node_network/virt-updating-node-network-config.html#virt-example-bridge-nncp_virt-updating-node-network-config).)
2. Create a Linux bridge NAD.
3. Configure the VM so that it recognizes the NAD. (I think this is more accurate than "attaching a VM". VMs don't get attached to anything. We configure them to recognize network devices. But if you think this is beyond the scope of your task, you can ignore my comment.)

Reply:
Done.